blocks/donation-form/class-give-donation-form-block.php

```diff
@@ -172,7 +186,17 @@ public function render_donation_form( $attributes ) {
 		$parameters['display_style']         = $attributes['displayStyle'];
 		$parameters['continue_button_title'] = trim( $attributes['continueButtonTitle'] );
 
-		return give_form_shortcode( $parameters );
+		$class_names = Html::classNames(
+			isset( $attributes['extUtilities'] ) ? $attributes['extUtilities'] : null,
+			! empty( $attributes['align'] ) ? 'align' . $attributes['align'] : null,
+			isset( $attributes['className'] ) ? $attributes['className'] : null
+		);
+
+		$content = '<div class="' . $class_names . '">';
```

Comment:
@mhimon Is there any reason to add this extra `div`?

Changes in DOM structure can cause arbitrary errors on the front. I would like to suggest you use [`give_form_wrap_classes`](https://github.com/impress-org/givewp/blob/78f7634b6da410d722196cb22922d79cd3c1961e/includes/class-give-donate-form.php#L876) filter to add custom classes to the donation form wrapper.

Reply:
ok, I will update the code. Thanks